docs/administration/garbage-collection/_index.md

```diff
@@ -32,6 +33,8 @@ To avoid triggering the garbage collection process too frequently, the availabil
     * **Daily**: Run garbage collection at midnight every day.
     * **Weekly**: Run garbage collection at midnight every Saturday.
     * **Custom**: Run garbage collection according to a `cron` job.
+    
+1. To delete untagged artifacts, select the check box **Delete Untagged Artifacts**. It's optional, the untagged artifacts will be GCed if selected.     
```

Comment:
please get rid of 'It's optional, the untagged artifacts will be GCed if selected.'